scripts/codestyle/run_uncrustify.sh

```diff
@@ -1,103 +1,68 @@
 #!/bin/bash
-if [ $# -gt 3 ]; then
-echo "Too many parameters specified"
-	exit 1
-fi
 
-VERBOSE=false
-HASHES=()
+TEMPDIR="out"
+UNCRUSTIFY_BASE=$(dirname $0)
+
+verbose="true"
+
+while getopts :qh opt; do
+	case $opt in
+	q)
+		verbose=""
+		;;
+	h)
+		echo "run_uncrustify.sh [options] [<git diff args>]"
+		echo ""
+		echo "Check codestyle of a change. Change generated by bypassing"
+		echo "diff-generating arguments to git-diff"
```

Comment:
Yeah, that's correct. Obviously, I spent not enough time in writing clear messages :-)
How about 
```
Check codestyle of a change. The change generated by passing
<git diff args> to git diff. No args means changes in worktree.
```